searchlib/src/vespa/searchlib/features/attributefeature.cpp

```diff
@@ -469,6 +472,9 @@ createTensorAttributeExecutor(const IAttributeVector *attribute, const vespalib:
     if (tensorType.is_dense()) {
         return stash.create<DenseTensorAttributeExecutor>(tensorAttribute);
     }
+    if (auto direct = dynamic_cast<const DirectTensorAttribute *>(tensorAttribute)) {
+        return stash.create<DirectTensorAttributeExecutor>(*direct);
+    }
```

Comment:
Consider reverting change to TensorAttributeExecutor now that we have a DIrectTensorAttributeExecutor ?

Reply:
yes - i forgot about that, thank you.